anime-list-master.xml

```diff
@@ -25738,6 +25739,9 @@
   </anime>
   <anime anidbid="9304" tvdbid="262954" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>JoJo no Kimyou na Bouken (2012)</name>
+    <mapping-list>
+      <mapping anidbseason="0" tvdbseason="0">;1-0;2-0;3-0;</mapping>
```

Comment:
I think instead of removing them, it'd be best to map this one to multiple and shift the blame to the client for not supporting more than 2 episodes 😇
```suggestion
      <mapping anidbseason="0" tvdbseason="1">;1-1+2+3+4+5+6+7+8+9;2-10+11+12+13+14+15+16+17+18+19;3-20+21+22+23+24+25+26;</mapping>
```
What do you think?

Reply:
In this case I think it's better to map these to 0 and let the client handle them as "TVDB has no idea about this"-episodes. These episodes are summaries and does not include the regular episodes in full. If a client were to map these to the regular episodes the poor user may think there are duplicates when it really is two different tellings of the same story.

Reply:
It says they're re-edits, so I assumed they'd be equivalent. But you make a good point, I was thinking of a "I got these *instead*" scenario, but if someone has both the regular episodes and these, it would get confusing.

I'm fine with either, actually. The Plex user will have to figure out an alternative anyways since files that match unmapped episodes don't show up at all but leaving them as they were would cause a conflict making them map to the wrong episodes.

Reply:
In that case I'd like to leave it like this and let the client/user deal with this corner case :slightly_smiling_face: